oraclesrv/tests/unittests/base.py

```diff
@@ -0,0 +1,57 @@
+import sys, os
+project_home = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../'))
```

Comment:
just a comment for the future with cookie cutter template: the sys.path is set for you by py.test config; this is no longer necessary (if `acutter` is used)